packages/camera/camera/README.md

```diff
@@ -66,6 +66,33 @@ As of version [0.5.0](https://github.com/flutter/plugins/blob/master/packages/ca
   }
 ```
 
+As of version [0.6.5](https://github.com/flutter/plugins/blob/master/packages/camera/camera/CHANGELOG.md#065) the startVideoRecording method can be used with the maxVideoDuration. To do this the result of the recording needs to be retrieved by calling controller.onCameraTimeLimitReachedEvent which accepts a callback to retrieve the XFile result. Like so:
+
+```dart
+    Future<void> startVideoRecording() async {
+      if (!controller.value.isInitialized) {
+        showInSnackBar('Error: select a camera first.');
+        return;
+      }
+  
+      if (controller.value.isRecordingVideo) {
+        // A recording is already started, do nothing.
+        return;
+      }
+  
+      try {
+        await controller.startVideoRecording(
+            maxVideoDuration: const Duration(milliseconds: 5000),
+        );
+        controller.onCameraTimeLimitReachedEvent(onCameraTimeLimitReached: (XFile file) {
+          //Handle the XFile
+        });
+      } on CameraException catch (e) {
+        _showCameraException(e);
+        return;
+      }
+    }
+```
```

Comment:
Really like that you didn't forget to updated the documentation in the README.md. But this should probably also be updated according the the discussion @BeMacized started on using a stream instead of registering a callback method.